templates/page.html

```diff
@@ -67,7 +96,23 @@
     {%- set toc = page.toc %}
     {%- endif %}
     {%- if toc %}
+    
     <div class="toc">
+      {%- if page.extra.section %}
+      <div class="series-toc">
+        <h4><a href="{{ tag.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}">{{ tag.name }} ({{ parts_string }})</a></h4>
+        <ol>
+        {%- for inner in tag.pages | sort(attribute="year") %}
+        {%- if inner.content == page.content %}
+        <li><a href="{{ inner.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}"><b>{{ inner.title }}</b></a></li>
+        {% else %}
+        <li><a href="{{ inner.permalink | safe }}{%- if uglyurls %}index.html{%- endif %}">{{ inner.title }}</a></li>
+        {%- endif %}
+        {%- endfor %}
+        </ol>
```

Comment:
On these lines below

Reply:
You'll have to block everything below the `<h4>`, to allow the title to be changed varying on whether it is toc or _in place_.